install/python-installer/scripts/hadoop_mods.py

```diff
@@ -122,10 +122,6 @@ def mod(self):
         }
         self.p.post('%s/config_groups' % cluster_url, hbase_config_group)
 
-        if dbcfgs['secure_hadoop'].upper() == 'Y':
```

Comment:
These are not values we are adding, just values we are trying to avoid deleting.  Since we are no longer writing value to hbase.coprocessor.region.classes, there is no danger of over-writing these security values.